bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/EvccHandler.java

```diff
@@ -0,0 +1,449 @@
+                        }
+                        break;
+                    case CHANNEL_LOADPOINT_TARGET_SOC:
+                        if (command instanceof QuantityType) {
+                            evccAPI.setTargetSoC(loadpoint, ((QuantityType<?>) command).intValue());
+                        }
+                        break;
+                    case CHANNEL_LOADPOINT_TARGET_TIME:
+                        if (command instanceof DateTimeType) {
+                            targetTimeZDT = ((DateTimeType) command).getZonedDateTime();
+                            ChannelUID channel = new ChannelUID(getThing().getUID(), "loadpoint" + loadpoint,
+                                    CHANNEL_LOADPOINT_TARGET_TIME);
+                            updateState(channel, new DateTimeType(targetTimeZDT));
+                            if (targetTimeEnabled) {
+                                try {
+                                    evccAPI.setTargetCharge(loadpoint, targetSoC, targetTimeZDT);
+                                } catch (DateTimeParseException e) {
+                                    logger.debug("Failed to set target charge", e);
+                                }
+                            }
+                        }
+                        break;
+                    case CHANNEL_LOADPOINT_TARGET_TIME_ENABLED:
+                        if (command == OnOffType.ON) {
+                            evccAPI.setTargetCharge(loadpoint, targetSoC, targetTimeZDT);
+                            targetTimeEnabled = true;
+                        } else {
+                            evccAPI.unsetTargetCharge(loadpoint);
+                            targetTimeEnabled = false;
+                        }
+                        break;
+                    case CHANNEL_LOADPOINT_PHASES:
+                        if (command instanceof DecimalType) {
+                            evccAPI.setPhases(loadpoint, ((DecimalType) command).intValue());
+                        }
+                        break;
+                    case CHANNEL_LOADPOINT_MIN_CURRENT:
+                        if (command instanceof QuantityType) {
+                            evccAPI.setMinCurrent(loadpoint, ((QuantityType<?>) command).intValue());
+                        }
+                        break;
+                    case CHANNEL_LOADPOINT_MAX_CURRENT:
+                        if (command instanceof QuantityType) {
+                            evccAPI.setMaxCurrent(loadpoint, ((QuantityType<?>) command).intValue());
+                        }
+                        break;
+                    default:
+                        return;
+                }
+            } catch (EvccApiException e) {
+                Throwable cause = e.getCause();
+                if (cause == null) {
+                    logger.debug("Failed to handle command {} for channel {}: {}", command.toString(), channelUID,
```

Comment:
`toString()` is redundant:
```suggestion
                    logger.debug("Failed to handle command {} for channel {}: {}", command, channelUID,
```